### DISCUSSION TITLE
e-commerce plugin

### DISCUSSION THREAD

**Discussion by mbnoimi** [wintercms/winter#649] [Q&A]
I want to create a new website for a small company with ability to sell customer service packages (very basic e-commerce) so I my choice was [Mall plugin](https://github.com/OFFLINE-GmbH/oc-mall-plugin) but I'm not pretty sure of comparability to Winter specially for the future as Winter marketplace still doesn't exist and I'm not good enough with Winter because I knew it since yesterday (I found some people in Reddit suggesting Winner as a good replacement to October after a shi*** experience)

I'm afraid that Winter will go completely in different direction which means -in my case- breaking my website because of incompatibility with October plugin,

What's your suggestion in my case?

**Comment by mjauvin** [comment]
I'm pretty biased on this, but no worries on our end, we're a team of 4 (more or less full-time), with more people joining in near future, currently maintaining/upgrading/improving the code base, so there is no single point of failure... that being said, there are never any garanties, even for frameworks that require a paid license.

If you're willing to join the community, you'll see it's pretty much alive!

Welcome!

**Comment by mjauvin** [comment]
Oh, and I didn't answer the e-commerce part, but we're working closely with Tobias Kuendig (OFFLINE/Mall plugin) to maintain compatibility with both Winter/October CMS.

Offline/Mall is my preferred choice as well.

**Reply by mbnoimi** [reply]
Awesome.
I'll follow your [Announcements](https://wintercms.com/blog/category/announcements) hoping to read the good news.

**Reply by mbnoimi** [reply]
I'm following the announcements since last Aug. Unfortunately I didn't read any progress on this matter!
Even Mall maintainer still releasing some fixes compatible with October while he didn't mention anything about Winter.

**Reply by damsfx** [reply]
> Even Mall maintainer still releasing some fixes compatible with October while he didn't mention anything about Winter.

The guys at OFFLINE GmbH do not plan to offer compatibility with WinterCMS.  
They made the choice to stay focused on OCMS.  
All the PRs to keep the compatibility with WinterCMS have been proposed by members of its community.  
Many of us use this plugin on our Winter sites, so we try to do everything we can to maintain this compatibility.  
Unfortunately we have reached a point where the v3 branch of the MALL plugin diverges too much to push changes.

At this time you must stay on the v2.x branch

**Reply by mbnoimi** [reply]
So sad for this decision. I didn't expect it!

**Reply by damsfx** [reply]
Stay connected ... Good news could always happen. 😉

**Comment by bennothommo** [comment]
To answer your question about compatibility, as time goes on we will deviate from October. They have their ideas and don't care about collaborating with us, so it will just happen. We are working hard to create our own ecosystem of plugins, and through the help of the community, we will try to maintain compatibility as long as possible.

All we can promise you is that the codebase is well looked after. As @mjauvin said, we all use this CMS for our own work and so we all are very motivated to keep it working flawlessly at peak performance.
